app/src/main/java/org/schabi/newpipe/player/mediasource/LoadedMediaSource.java

```diff
@@ -83,6 +85,38 @@ public void removeEventListener(final MediaSourceEventListener eventListener) {
         source.removeEventListener(eventListener);
     }
 
+    /**
+     * Adds a {@link DrmSessionEventListener} to the list of listeners which are notified of DRM
+     * events for this media source.
+     *
+     * @param handler       A handler on the which listener events will be posted.
+     * @param eventListener The listener to be added.
+     */
+    @Override
+    public void addDrmEventListener(final Handler handler,
+                                    final DrmSessionEventListener eventListener) {
+        source.addDrmEventListener(handler, eventListener);
+    }
+
+    /**
+     * Removes a {@link DrmSessionEventListener} from the list of listeners which are notified of
+     * DRM events for this media source.
+     *
+     * @param eventListener The listener to be removed.
+     */
+    @Override
+    public void removeDrmEventListener(final DrmSessionEventListener eventListener) {
+        source.removeDrmEventListener(eventListener);
+    }
+
+    /**
+     * Returns the {@link MediaItem} whose media is provided by the source.
+     */
+    @Override
+    public MediaItem getMediaItem() {
+        return MediaItem.fromUri(stream.getUrl());
```

Comment:
the **MediaItem** already exists in the `source` object, this should be `return source.getMediaItem();`